src/mixxx.h

```diff
@@ -166,6 +168,9 @@ class MixxxMainWindow : public QMainWindow {
                           const QString& translationPath, QTranslator* pTranslator);
     void checkDirectRendering();
     bool confirmExit();
+    
+    void linkSkinWidget(ControlObjectSlave** pCOS, 
```

Comment:
Yes, we usually follow the Qt approach that a reference parameter should be always const. 
This is because in C++ you cannot see in the calling code, that the parameter passed to a function can be changed after the function call. 
linkSkinWidget(m_pCOShowSamplers, ... can be a pass by value or a pass by reference. If we can assume all references are const, we do not need to care about the difference in the calling code.  
Instead a for a linkSkinWidget(&m_pCOShowSamplers, ... you see at once that the called function is able to change m_pCOShowSamplers

From the resulting machine code both solutions should be equivalent.